rest-service/manager_rest/rest/resources_v3_1/blueprints.py

```diff
@@ -72,3 +75,23 @@ def put(self, blueprint_id, **kwargs):
         return UploadedBlueprintsManager().\
             receive_uploaded_data(data_id=blueprint_id,
                                   visibility=visibility)
+
+    @swagger.operation(
```

Comment:
I didn't realise we were missing this in swagger before. Oh dear. We should probably have some tests checking that all of our API paths are actually in swagger?

Reply:
(but not as part of this)

Reply:
Agree, also there should be work on updating the swagger docs and fixing our swagger specification generation